spec/service_models/miq_ae_service_ems_cluster_spec.rb

```diff
@@ -1,10 +1,33 @@
 describe MiqAeMethodService::MiqAeServiceEmsCluster do
-  let(:cluster) { FactoryBot.create(:ems_cluster) }
+  let(:cluster) { FactoryBot.create(:ems_cluster, :ext_management_system => FactoryBot.create(:ext_management_system)) }
+  let(:host) { FactoryBot.create(:host) }
   let(:svc_cluster) { MiqAeMethodService::MiqAeServiceEmsCluster.find(cluster.id) }
 
   it "#show_url" do
     ui_url = stub_remote_ui_url
 
     expect(svc_cluster.show_url).to eq("#{ui_url}/ems_cluster/show/#{cluster.id}")
   end
+
+  before do
```

Comment:
Seems this `before` block is not needed and the `@base_queue_options` setup can go right into the test since it is never reused.